services/web/project/blueprint/metrics_blueprint.py

```diff
@@ -30,4 +30,11 @@ def calculate_shared_pulls(login):
     start_time = time.time()
     semantic_metrics_service.save_shared_pulls(login)
     print("--- shared_pulls time:  %s seconds ---" % (time.time() - start_time))
+    return '', 200
+
+@bp_metrics.route('/metrics/compile/<login>', methods=['POST'])
+def compile(login):
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [Redefining built-in 'compile'](https://app.codacy.com/manual/mpgabriel95/TCC/pullRequest?prid=5653140)